tests/components/airzone/test_coordinator.py

```diff
@@ -0,0 +1,35 @@
+"""Define tests for the Airzone coordinator."""
+
+from unittest.mock import MagicMock, patch
+
+from aiohttp import ClientConnectorError
+
+from homeassistant.components.airzone.const import DOMAIN
+from homeassistant.components.airzone.coordinator import SCAN_INTERVAL
+from homeassistant.core import HomeAssistant
+from homeassistant.util.dt import utcnow
+
+from .util import CONFIG, HVAC_MOCK
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+
+async def test_coordinator_client_connector_error(hass: HomeAssistant):
+    """Test ClientConnectorError on coordinator update."""
+
+    entry = MockConfigEntry(domain=DOMAIN, data=CONFIG)
+    entry.add_to_hass(hass)
+
+    with patch(
+        "aioairzone.localapi_device.AirzoneLocalApi.get_hvac",
```

Comment:
Please patch the module where its imported instead of broadly

Reply:
@bdraco I'm sorry but I don't think I know what you want me to do. Could you provide an example?

Reply:
If the instance you want to patch is imported in `homeassistant.components.airzone` then patch  `homeassistant.components.airzone.AirzoneLocalApi`... instead.

Reply:
Ah ok, sorry!
I did it this way on the rest of the tests…
Should I change it in the future for the rest of the tests too?

Reply:
Yes, we always want to keep the patch as narrow as possible

Reply:
> If the instance you want to patch is imported in `homeassistant.components.airzone`, patch `homeassistant.components.airzone.AirzoneLocalApi`... instead.

Changed in 2c08b50cee